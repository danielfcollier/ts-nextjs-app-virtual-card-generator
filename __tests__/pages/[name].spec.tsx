import { render } from '@testing-library/react';
import History from '@/pages/[name]';

it('Snapshot of Card [name] page unchanged', () => {
  const { container } = render(<History />);
  expect(container).toMatchSnapshot();
});
