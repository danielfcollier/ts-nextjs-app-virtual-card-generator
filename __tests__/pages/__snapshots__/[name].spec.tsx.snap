// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of Card [name] page unchanged 1`] = `
<div>
  <div
    class="card-container"
  >
    <p>
      Hello, my name is 
    </p>
    <h1>
      My history
    </h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur
    </p>
    <div
      class="container"
    >
      <div
        class="row row row-cols-sm-6 row-cols-2"
      />
    </div>
  </div>
</div>
`;
